src/screens/MainScreen.js

```diff
@@ -112,11 +113,13 @@ class MainScreen extends React.Component {
     const renderTxHistory = () => {
       if (this.props.txList && (this.props.txList.length > 0)) {
         return (
-          <TxHistoryView
-            txList={this.props.txList}
-            token={this.props.selectedToken}
-            onTxPress={this.onTxPress}
-          />
+          <ScrollView>
```

Comment:
Should it be inside `<TxHistoryView ...>`?

Reply:
@msbrogli  eu acredito que não, porque o <TxHistoryView...> é uma tag unica. Honestamente, aprendendo react-native para ajudar no projeto, entao não sei muita coisa ainda ^^ .

se tiver uma forma de realizar isso e vc puder me indicar um link eu vejo como acertar.

[English]
@msbrogli I don't think so, because the <TxHistoryView...> is a unique tag. Honestly, learning react-native to help with the project, so I don't know much yet ^^ .

if there is a way to do this and you can show me a link I see how to get it right.